app/src/main/java/org/p2p/wallet/swap/jupiter/statemanager/SwapStateManagerHolder.kt

```diff
@@ -0,0 +1,18 @@
+package org.p2p.wallet.swap.jupiter.statemanager
+
+object SwapStateManagerHolder {
```

Comment:
can be class, but singleOf in koin

Reply:
i do remind you that we are planning to write unit tests, and singletons via object are not preferred